core/schema/container.graphqls

```diff
@@ -27,10 +27,17 @@ type Container {
   build(context: DirectoryID!, dockerfile: String): Container!
 
   "This container's root filesystem. Mounts are not included."
-  fs: Directory!
+  rootfs: Directory!
+
+  "This container's root filesystem. Mounts are not included."
+  fs: Directory! @deprecated(reason: "replaced by rootfs")
+
+  "Initialize this container from this DirectoryID"
+  withRootfs(id: DirectoryID!): Container!
 
   "Initialize this container from this DirectoryID"
   withFS(id: DirectoryID!): Container!
+    @deprecated(reason: "replaced by withRootfs")
```

Comment:
Nice! Will do

Reply:
Just noticed an issue... Since the reason references a GraphQL name, when you codegen as is you don't get it through the proper naming conventions.

Reply:
Yeah, I noticed that as well when doing this.

I was thinking having a somewhat "wrong" name is better than not suggesting an alternative at all. And instead of picking the name in a given SDK, I just picked the GraphQL field name. WDYT?

Reply:
Implemented the suggestion above in the last commit -- @helderco thoughts?

Reply:
I like that it enables me to format and turn it into a link, but I'm wondering about the playground. If you go to the GraphQL API directly the `@` will look alien there.

Maybe if we have a set of formatted strings: `Replaced by <field>.` and any others that we may need. So an SDK regex could match against `Replaced by withRootfs.` and extract the field from that pattern.

Reply:
> I like that it enables me to format and turn it into a link, but I'm wondering about the playground. If you go to the GraphQL API directly the @ will look alien there.

Not *too* alien I think? Maybe \`field\` would look more natural?

> Maybe if we have a set of formatted strings: Replaced by <field>. and any others that we may need. So an SDK regex could match against Replaced by withRootfs. and extract the field from that pattern.

Not sure I fully understand this. What would we write as the reason in GraphQL and how would we know which field it is?

Reply:
Changed this to

```
Replaced by `withRootFs`
```

Looks just fine in raw GraphQL

Reply:
> Not sure I fully understand this. What would we write as the reason in GraphQL and how would we know which field it is?

In GraphQL it would be just `Replaced by withRootfs`. Then in the SDKs you would match by the whole phrase like `Replaced by (.*).` instead of just the subject.

Anyway, I'm good with the backticks, I'm going to update Python now quickly so you don't get blocked on it.

Reply:
Oh, I see!

Well, I assumed this could be more dynamic. For instance, we're about to deprecate `Host().Workdir()`. Depreciation reason could be:

```
Use `directory` with `path` set to . instead.
```

Reply:
To be able to cross-reference in python it's simpler to assume backticks only for other **methods**:

```
Use `directory` with path set to '.' instead.
```
In another object:
```
Use `Container.directory` instead.
```